control-plane-operator/controllers/hostedcontrolplane/v2/kas/config.go

```diff
@@ -117,7 +117,7 @@ func generateConfig(p KubeAPIServerConfigParams) (*kcpv1.KubeAPIServerConfig, er
 									Kind:       "PodSecurityConfiguration",
 								},
 								Defaults: podsecurityadmissionv1.PodSecurityDefaults{
-									Enforce:        "restricted",
+									Enforce:        "privileged",
```

Comment:
I think that we need updates to the test fixtures too which I would think verify that we are indeed rendering the proper default PSA config for 4.20.

Reply:
I also thought this but Bryce ran `go test` with `UPDATE=true` and there was no diff.  Also `unit` didn't fail so :man_shrugging:  I guess not.

Reply:
I wonder if an update to the OpenShift API go module that contains the default change would yield a unit test failure.